src/classes/client.ts

```diff
@@ -76,21 +78,29 @@ export class AIHordeClient extends Client {
 	}
 
 	async loadHordeStyles() {
-		const source = this.config.data_sources?.styles_source ?? `https://raw.githubusercontent.com/db0/Stable-Horde-Styles/main/styles.json`
+		const source = this.config.data_sources?.styles_source || `https://raw.githubusercontent.com/AI-Horde/Stable-Horde-Styles/main/styles.json`
```

Comment:
please make sure that the link is actually valid, in this case it is not valid and returns a 404